frontend/src/components/Search/SimpleSearch/SimpleSearch.logic.ts

```diff
@@ -0,0 +1,49 @@
+import type { QueryParams } from "~/stores/searchParams";
+import { DocumentKind } from "~/types";
+
+export function getUrl(category?: string) {
+  if (category?.startsWith(DocumentKind.CaseLaw)) {
+    return `/v1/case-law`;
+  } else if (category?.startsWith(DocumentKind.Norm)) {
+    return `/v1/legislation`;
+  } else {
+    return `/v1/document`;
+  }
+}
+
+/**
+ * Converts the params used in the pinia store to URL params as expected by
+ * the API.
```

Comment:
I think the function is self explanatory, it wouldn't need the comment. Also for the comments below. The code to me looks clean

Reply:
Okay, fair point :)